### DISCUSSION TITLE
Authentication issue

### DISCUSSION THREAD

**Discussion by r8ders2k** [dgreif/ring#1162] [Q&A]
Anyone having issue with the Homebridge not being able to authenticate with Google/Nest? As my setup was working fine, but then I saw this in my Homebridge log...

<img width="1247" alt="Screenshot 2023-02-17 at 4 20 54 PM" src="https://user-images.githubusercontent.com/60878009/219820256-0f7f1a82-80eb-4a41-8809-87f1daa914da.png">

and I have tried getting a new "refreshToken", but still getting the same authentication error.

**Comment by dgreif** [comment]
This repo is not associated with Google/Nest in any way. You may be looking for https://github.com/chrisjshull/homebridge-nest?
